README.md

```diff
@@ -76,10 +77,40 @@ providing an API to convert user-entered number and date strings - in their
 own format - into actual numbers and dates, and conversely, to format numbers
 and dates into that string format.
 
+Even if the application deals only with the English locale, it may still need
+globalization to format programming language bytes into human-understandable
+language and vice-versa in an effective and reasonable way. For example, to
+display something better than "Edited 1 minutes ago".
+
+<a name="why_globalize"></a>
+### Why Globalize?
+
+Globalize provides number formatting and parsing, date and time formatting and
+parsing, currency formatting, message formatting (ICU message format pattern),
+and plural support.
+
+Design Goals.
+
+- Leverage the official CLDR JSON data.
+- Allow users to load as much or as little data as they need.
+- Avoid duplicating data if using multiple i18n libraries that leverage CLDR.
+- Run in browsers and node.js.
+- Make globalization as easy to use as jQuery.
+
+Strengths of Using Globalize.
+
+- Based on CLDR data (largest and most extensive i18n repository).
+- No bugs due to outdated i18n content (it has no hard-coded i18n data).
+- Consistency. Same results are obtained across browsers and on the server.
```

Comment:
@jzaefferer,

There are two comparisons that are important to emphasize. One, getting consistent results between different browsers. Two, getting consistent results between a browser and the server.

After adding "and on the server" to the sentence ("Same results are obtained across browsers and on the server"), the second emphasis is privileged.

The below, although repetitive, makes it clear. Can we improve it?

"- Consistent. Same results are obtained across browsers. Same results are obtained between browsers and the server."